conjure_oxide/Cargo.toml

```diff
@@ -39,6 +39,7 @@ default = ["extra-rule-checks"]
 extra-rule-checks = ["conjure_core/extra-rule-checks"]
 unstable = []
 unstable-solver-interface = ["unstable"]
+skip_native_parser = true
```

Comment:
I think this needs to be in (some) config.toml files under the integration directory instead.

Reply:
I'm not sure where to add this. I don't see another cargo.toml under any of the folders in the tests directory.

Reply:
it's not cargo.toml, it's config.toml. only exists for some tests, if it doesn't exist the default values are used.